packages/scandipwa/src/component/Loader/Loader.style.scss

```diff
@@ -40,3 +40,13 @@
         }
     }
 }
+
+#root {
+    div[class^='LocalizationWrapper'] {
```

Comment:
LocalizationWrapper is unreliable, since it's added by a plugin and can be disabled.

Reply:
Maybe you can add some additional prop to the loaded that has to be fixed, and then convert this prop to a class with `mods={ { isFixed } }`?

Reply:
removed that loader altogether since it's interchangeable with Router-Loader